GoalMsg mission_goal
string mission_name
bool doRunwayMission
